README.md

```diff
@@ -144,6 +144,12 @@ Sharing in Tomcat.
 
 See instructions [here](http://enable-cors.org/server_tomcat.html) for Tomcat 7 and [here](https://www.dforge.net/2013/09/16/enabling-cors-on-apache-tomcat-6/) for Tomcat 6.
 
+## Usage
+Before using the Order Entry Open Web Application, the administrator should make the following one time configurations using the admin portal. 
+1. Create an Encounter Type called `Order Entry` or any other name.
+2. Create a global property `order.encounterType` whose value should correspond to the name given to the Encounter Type created above.
```

Comment:
Global property or setting?

Reply:
@dkayiwa, I am wondering why we have to change the readMe and error message to setting, yet in the administration UI, it appears as global property?

Reply:
Which UI? Can you point me to it on the demo server?

Reply:
That needs to be changed to settings

Reply:
Noted, I will make the necessary changes now.